docs/benchmarks/data-caching.md

```diff
@@ -73,43 +58,47 @@ The client can simultaneously use multiple servers, one server with several ip a
     IP_ADDRESS_VM2, 11211
     ...
 
-You can use the `vim` command for modifying this file inside the container.
+
+
+To start the client container use the following command:
+
+    $ docker run -idt --name dc-client --net host -v PATH_TO_DOCKER_SERVERS_FOLDER:/usr/src/memcached/memcached_client/docker_servers/ cloudsuite/data-caching:client
```

Comment:
If the container is running as a daemon `-d`, we don't have to make it interactive `-it`. Just leaving `-d` is enough.

Reply:
After trial, I think we don't need `-i`, but `-t` is important to keep the container alive.

Reply:
I recommend keeping `-i` because a user may want to enter the container later and modify it manually for any reason.